pages/index.tsx

```diff
@@ -203,12 +209,25 @@ export default function Home({ metricsData, siteConfigData }) {
   )
 }
 
+
 export const getStaticProps: GetStaticProps = async () => {
   const siteConfigData = siteConfig
   const { social } = siteConfig
-  const metricsData = metrics
+  const { volumeUsd } = await cowSdk.cowSubgraphApi.getTotals()
+  
+  const metricsData = [
+    {label: "Total Volume", value: numberFormatter.format(+volumeUsd) + '+'},
+    
+    {label: "All Time Trades", value: "321K+"},
+
+    // https://dune.xyz/gnosis.protocol/GPv2-Trader-Surplus
+    //  Resonable + Unusual
+    {label: "Surplus generated for users", value: "$64M+"}
+  ]
```

Comment:
We recently got an alpha version of the Dune API. This version allows us to query for results by Query ID although it does not allow us to refresh the computation. We could already fill the "All Time Trades" count here via the API service as it would always be strictly increasing. The query itself if refreshed whenever someone visits the dashboard. So, if we assume that at least once per day, someone is checking the dashboard, then the value would always be accurate within the last 24 hours. Is this something you might want to use here?

For more details, here are the docs - https://api.dune.com/docs#/

and here is the query we could retrieve from: https://dune.com/queries/37064

cc @anxo and @alfetopito

Reply:
Example request for total trades:
```
curl -X 'GET' \
  'https://api.dune.com/api/v0/query/1034337/results' \
  -H 'accept: application/json' \
  -H 'X-DUNE-API-KEY: BIG-SECRET'
```

Which gives response:

```
{
  "metadata": {
    "executed_at": "2022-07-13T07:04:03.561808Z",
    "job_id": "22938a8e-73f7-4afe-a243-1000ecb41c57",
    "query_duration_millis": 1000,
    "query_id": 1034337,
    "query_version": 1,
    "result_bytes": 18,
    "result_rows": 1
  },
  "column_names": [
    "count"
  ],
  "rows": [
    {
      "count": 325364
    }
  ]
}
```